### PR TITLE
Add/update some CCF B Computer Architecture/Parallel Programming/Stor…

### PR DESCRIPTION
**Pull request by stbyu**
…age Technology conferences.

 

### Which conference does this PR update?

-  cgo 25
- cluster 24
- fast 25
- iccd 24
- ipdps 24
- pact 24
- spaa 24

### Related URL

- https://conf.researchr.org/home/cgo-2025
- https://clustercomp.org/2024
- https://www.usenix.org/conference/fast25
- https://www.iccd-conf.com/Call_for_papers.html
- https://www.ipdps.org/
- https://pact2024.github.io/
- https://spaa.acm.org/

### CODE REVIEW

**Comment by jacklightChen** [comment]
image
plz check the format errors in some of the ymls~

File: conference/DS/fast.yml [hunk 1]
```diff
@@ -36,3 +36,11 @@
       timezone: UTC-7
       date: FEBRUARY 27-29, 2024
       place: SANTA CLARA, CA, USA
+    - year: 2025
+      id: fast25
+      link: https://www.usenix.org/conference/fast25
+      timeline:
+        - deadline: '2024-09-17 23:59:59'
+      timezone: UTC-7
+      date: FEBRUARY 25-27, 2024
```
Comment (jacklightChen) [inline conference/DS/fast.yml:45]: 2025？

File: conference/DS/iccd.yml [hunk 2]
```diff
@@ -14,3 +14,13 @@
       timezone: AoE
       date: November 6-8, 2023
       place: Washington DC, USA
+    - year: 2024
+      id: iccd24
+      link: https://www.iccd-conf.com/Call_for_papers.html
+      timeline:
+        - abstract_deadline: '2024-05-20 23:59:00'
+          deadline: '2023-05-20 23:59:00'
```
Comment (jacklightChen) [inline conference/DS/iccd.yml:22]: seems to be 2024

**Review by jacklightChen — CHANGES_REQUESTED** [review]

File: conference/DS/iccd.yml (hunk 2 above)
Comment (stbyu) [reply conference/DS/iccd.yml:22]: Yes, sorry, you are right, I will modify them. Thank you

**Review by jacklightChen — APPROVED** [review]
LGTM

**Pull request merged**